nornir_nautobot/plugins/inventory/nautobot.py

```diff
@@ -151,13 +151,19 @@ def load(self) -> Inventory:
             host["name"] = device.name or str(device.id)
             host["groups"] = []
 
+            # Check to see if a platform has been defined
+            if hasattr(device.platform, "slug"):
```

Comment:
I'll leave it up to you, but I believe you can just use `getattr(device.platform, "slug", None)` if you'd like, but same thing. You could even just move it to L166.

Reply:
I like that simplicity. I did a small bit of research. This pattern with `getattr` is now engrained into my mind and will use.